api/lib/application/pre-response-utils.js

```diff
@@ -14,9 +16,17 @@ function handleDomainAndHttpErrors(request, h) {
     return errorManager.handle(request, h, new UnauthorizedError(undefined, 401));
   }
 
+  if (_isError5XX(response) && config.logging.isError5XXLoggingEnabled) {
+    logger.error(response);
```

Comment:
Est-ce que cela n'emêche pas la propagation à Sentry ?
Cela peut être utile de le garder pour du dev avec hapi-sentry par exemple.
Dans ce cas, on peut l'expliciter ainsi

```suggestion
    logger.error(response);
    passToExternalErrorMonitoring({error : response});
    
    const passToExternalErrorMonitoring({error})=>{
       throw(error)
    }
```

Reply:
Bonne question, je serais étonné que ça change quelque-chose pour Sentry, il ne devrait pas avoir connaissance du fait qu'on ajoute un log ou non. De plus en prod on ne passe pas dans le if, à moins de l'activer.
Je vais vérifier en activant Sentry en local pour être certain.

Reply:
AFAIK Sentry catch les erreurs non catchées, et ici on ne les throw plus, donc il ne les verra plus

Reply:
Correction: il semble que Sentry passe par les évènements..
https://github.com/hydra-newmedia/hapi-sentry/blob/master/index.js#L62

Reply:
J'ai testé, ça fonctionne bien avec Sentry en l'état. Note : au niveau de ce code il ne s'agit plus d'exeption mais de response, qui suit son chemin dans Hapi.